docs/getting-started.md

```diff
@@ -0,0 +1,123 @@
+# Dock Wallet SDK
+
+This guide walks you through the process of setting up the Dock Wallet SDK, creating a wallet, managing decentralized identifiers (DIDs), adding credentials, and verifying them.
+
+## Installation
+
+To start, you need to install the necessary packages for the wallet SDK and the data store. Open your terminal and run the following command:
+
+```bash
+yarn add @docknetwork/wallet-sdk-core @docknetwork/wallet-sdk-data-store-typeorm
+```
+
+The `@docknetwork/wallet-sdk-core` provides the core wallet functionality, while `@docknetwork/wallet-sdk-data-store-typeorm` handles data persistence using a local SQLite database.
```

Comment:
should we talk about using the cloud wallet storage somewhere here?

Reply:
I've added that to line 33

Reply:
https://github.com/docknetwork/react-native-sdk/pull/297/files#diff-31bcba2ccafa41d46fbbd6d1219f7f1e3b1fb3cad9faa8e4dc521bbb579dd7b3R33